src/test/ui/ifmt.rs

```diff
@@ -144,10 +144,10 @@ pub fn main() {
     t!(format!("{:e}", 1.2345e6f32), "1.2345e6");
     t!(format!("{:e}", 1.2345e6f64), "1.2345e6");
     t!(format!("{:E}", 1.2345e6f64), "1.2345E6");
-    t!(format!("{:.3e}", 1.2345e6f64), "1.234e6");
-    t!(format!("{:10.3e}", 1.2345e6f64),   "   1.234e6");
-    t!(format!("{:+10.3e}", 1.2345e6f64),  "  +1.234e6");
-    t!(format!("{:+10.3e}", -1.2345e6f64), "  -1.234e6");
+    t!(format!("{:.3e}", 1.2345e6f64), "1.235e6");
+    t!(format!("{:10.3e}", 1.2345e6f64),   "   1.235e6");
+    t!(format!("{:+10.3e}", 1.2345e6f64),  "  +1.235e6");
+    t!(format!("{:+10.3e}", -1.2345e6f64), "  -1.235e6");
```

Comment:
It looks to me like these were correct before. Round-to-even on 1234.5 should go to 1234, not 1235.